modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointHistoryResult.java

```diff
@@ -58,9 +59,11 @@ public CheckpointHistoryResult(
     }
 
     /**
-     * @return Reserved checkpoint.
+     * Getting the oldest reserved checkpoint marker.
```

Comment:
The common practice is using the Present Simple for writing Javadoc. `Returns the oldest reserved checkpoint marker` looks more natural to me.